test/Interop/Cxx/stdlib/use-std-span.swift

```diff
@@ -176,6 +176,18 @@ StdSpanTestSuite.test("Init SpanOfInt") {
   expectFalse(cs.empty())
 }
 
+StdSpanTestSuite.test("Init SpanOfInt from Swift array") {
```

Comment:
Since these changes aren't benchmarks, could you please extract them to a separate PR?

Reply:
Ok sure!